set.mm

```diff
@@ -471102,6 +471246,54 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
   $}
 
 
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+    Transpositions
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  ${
+    pmtridf1o.a $e |- ( ph -> A e. V ) $.
+    pmtridf1o.x $e |- ( ph -> X e. A ) $.
+    pmtridf1o.y $e |- ( ph -> Y e. A ) $.
+    pmtridf1o.t $e |- T = if ( X = Y , ( _I |` A )
+                                          , ( ( pmTrsp ` A ) ` { X , Y } ) ) $.
+    $( Transpositions of ` X ` and ` Y ` , extended with the identity when they
+       are equal, are bijections.  (Contributed by Thierry Arnoux,
+       1-Jan-2022.) $)
+    pmtridf1o $p |- ( ph -> T : A -1-1-onto-> A ) $=
+      ( wf1o wceq wa cfv syl5eq wcel syl adantr syl3anc eqid cid cres cpr cpmtr
+      cif iftrue adantl f1oi a1i f1oeq1 biimpar syl2anc wne crn wn simpr neneqd
+      iffalse wss c2o cen wbr prssd pr2nelem pmtrrn eqeltrd pmtrff1o pm2.61dane
+      ) ABBCKZEFAEFLZMZCUABUBZLZBBVLKZVIVKCVJVLEFUCZBUDNZNZUEZVLJVJVRVLLAVJVLVQ
+      UFUGOVNVKBUHUIVMVIVNBBCVLUJUKULAEFUMZMZCVPUNZPVIVTCVQWAVTCVRVQJVTVJUOVRVQ
+      LVTEFAVSUPZUQVJVLVQURQOVTBDPZVOBUSVOUTVAVBZVQWAPAWCVSGRVTEFBAEBPZVSHRZAFB
+      PZVSIRZVCVTWEWGVSWDWFWHWBEFBBVDSBVOWAVPDVPTZWATZVESVFBWAVPCWIWJVGQVH $.
+
+    $( Value at ` X ` of the transposition of ` X ` and ` Y ` , extended with
+       the identity when they are equal.  (Contributed by Thierry Arnoux,
```

Comment:
Maybe: Value at ` X ` of the transposition of ` X ` and ` Y ` (understood to be the identity when ` X = Y ` ) ?

Reply:
Ok!